### PR TITLE
PageMetaData component

### PR DESCRIPTION
**Pull request by corwintines**
- create PageMetaData component
- add to home, downloads, and docs pages
- create constants needed for component

## TODO
- [x] homepage description
- [x] downloads description

### CODE REVIEW

**Comment by wackerow** [comment]
Put up two commits with some small clean-up, and added descriptions for the home and downloads pages. 

Feel free to comment on suggestions for the descriptions.

**Review by wackerow — APPROVED** [review]

File: src/components/UI/PageMetadata.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+interface Props {
+  title: string;
+  description: string;
+  image?: string;
+}
+
+export const PageMetadata: React.FC<Props> = ({ title, description, image }) => {
+  const router = useRouter();
+  const url = `${SITE_URL}${router.asPath}`;
+  const fullTitle = `${title} | ${SITE_NAME}`;
```
Comment (samajammin) [inline src/components/UI/PageMetadata.tsx:15]: How to handle if no `title` and/or `description` is passed? Should we have a default, or hide those properties?

Comment (wackerow) [reply src/components/UI/PageMetadata.tsx:15]: They're currently both required values per the interface above. This is the same pattern we use on the Blog, but on E.org we do make the description optional, and provide a fallback. In both of those cases we require a `title` which I think is appropriate.

If we'd prefer, we could make `description` optional and provide a fallback, but might suggest we leave it as required so the description it most likely to make sense for each page.

**Review by nhsz — APPROVED** [review]
Thanks, LGTM! ⚡️

**Pull request merged**